front/types/document.ts

```diff
@@ -1,17 +0,0 @@
-export type DocumentType = {
```

Comment:
Shouldn't we keep the two types sperated? It feels not OK to expose a DustAPI* type as return of our front APIs?